### ISSUE TITLE
pr-rebase-needed fails, requiring repoToken

### ISSUE THREAD

**Issue by caugner** [mdn/bob#849]
See: https://github.com/mdn/bob/runs/7954172846?check_suite_focus=true

```
Run eps1lon/actions-label-merge-conflict@releases/2.x
  with:
    dirtyLabel: rebase needed :construction:
    commentOnDirty: This pull request has merge conflicts that must be resolved before it can be merged.
Error: Error: Input required and not supplied: repoToken
Error: Input required and not supplied: repoToken
```

**Comment by caugner** [comment]
@schalkneethling Any ideas what might cause this?

**Comment by caugner** [comment]
I have resolved this in 9066adc647e9c272327830dbc0f20c4f8840e572 by using auto-generated GITHUB_TOKEN instead of GH_TOKEN.
